Gems/Vegetation/Code/Source/Editor/EditorSpawnerComponent.h

```diff
@@ -29,6 +29,6 @@ namespace Vegetation
         static constexpr const char* const s_componentDescription = "Creates dynamic vegetation in a specified area";
         static constexpr const char* const s_icon = "Editor/Icons/Components/Vegetation.svg";
         static constexpr const char* const s_viewportIcon = "Editor/Icons/Components/Viewport/Vegetation.svg";
-        static constexpr const char* const s_helpUrl = "https://o3de.org/docs/user-guide/components/reference/vegetation-layer-spawner/";
+        static constexpr const char* const s_helpUrl = "https://o3de.org/docs/user-guide/components/reference/vegetation/layer-spawner/";
```

Comment:
seems unrelated to the migration, but I assume this rides along with a merge or something

Reply:
yeah it was for a sign-off command I had to do to pass the sign off check